src/components/Screen/index.js

```diff
@@ -74,26 +75,38 @@ export class Screen extends Component {
 				</Header.Content>
 				<Header.Actions>
 					<Header.Action
-						title={notificationsEnabled ? 'Disable notifications' : 'Enable notifications'}
+						title={notificationsEnabled ? I18n.t('Disable notifications') : I18n.t('Enable notifications')}
 						onClick={notificationsEnabled ? this.triggerDisableNotifications : this.triggerEnableNotifications}
 					>
-						{notificationsEnabled ?
-							<NotificationsEnabledIcon width={20} /> :
-							<NotificationsDisabledIcon width={20} />
-						}
+						<Tooltip.Container>
```

Comment:
You just need a `Tooltip.Container` wrapping everything that has a tooltip, e.g.:

```jsx
<Tooltip.Container>
  <Header.Actions>
    // Everything, including Tooltip.Trigger
  </Header.Actions>
</Tooltip.Container>
```